cli/engine/ApplyEngine.py

```diff
@@ -123,8 +123,8 @@ def assert_no_master_downscale(self):
                     raise Exception("ControlPlane downscale is not supported yet. Please revert your 'kubernetes_master' count to previous value or increase it to scale up Kubernetes.")
 
     def assert_no_postgres_nodes_number_change(self):
+        self.process_configuration_docs()
```

Comment:
We should not process al configuration documents at this point here jet. Check input documents if 'configuration/feature-mapping' is present, if not take from default and maybe merge.